legal-api/setup.cfg

```diff
@@ -72,7 +72,7 @@ max_line_length=120
 notes=FIXME,XXX,TODO
 ignored-modules=flask_sqlalchemy,sqlalchemy,SQLAlchemy,alembic,scoped_session
 ignored-classes=scoped_session
-disable=C0301,W0511,W0703,R0801,R0902,R0401,R1718
+disable=C0301,W0511,W0703,R0801,R0902,R0401,R1718,E0605
```

Comment:
E0605 shouldn't be added

Reply:
@severinbeauvais Those lint errors can be fixed adding a comma to make it a tuple: 
```
__all__ = ('API',)
```

Reply:
Removed in next commit.

Reply:
I think that now you can rebase it. I think I fixed all of them in a previous change.

Reply:
Rebased in next commit. (Lint error was indeed fixed.)